### PR TITLE
feat: Add "HamburgerMenu" and "Add" icons

### PR DESCRIPTION
**Pull request by rogervera**
See:
https://www.figma.com/file/LHkckE7icSpNMdQUxcYRg1/Easy-v2?node-id=6733%3A41539

### CODE REVIEW

File: src/icons/Add.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import type { IconProps } from './types'
+
+const Add = ({ color = '#fff', scale = 1, strokeWidth = 2 }: IconProps) => (
```
Comment (jackrr) [inline src/icons/Add.tsx:4]: :grey_question: Hmmm, should color control the `fill` on `rect`? Seems like that's more likely the color that a consumer of the icon will want to change.

File: src/icons/HamburgerMenu.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import type { IconProps } from './types'
+
+const HamburgerMenu = ({ scale = 1 }: IconProps) => (
```
Comment (jackrr) [inline src/icons/HamburgerMenu.tsx:4]: :bulb: If "MENU" and the three bars can be the same color, this could support a color prop as well.

**Review by jackrr — APPROVED** [review]

File: src/icons/Add.tsx (hunk 1 above)
Comment (rogervera) [reply src/icons/Add.tsx:4]: Ah, I prefer that.

File: src/icons/HamburgerMenu.tsx (hunk 2 above)
Comment (rogervera) [reply src/icons/HamburgerMenu.tsx:4]: Leaving this as is for now. Perhaps to be revisited on QA!

**Pull request merged**